example/clirunners/0_0_build.bat

```diff
@@ -1,3 +1,2 @@
 echo "Compiling the test code..."
-cd ..
-mvn clean compile
\ No newline at end of file
+mvn -f ../pom.xml clean compile
```

Comment:
@other019 Please double check if it's not better to run "call mvn -f ../pom.xml clean compile". I rely on your decision.

Reply:
https://stackoverflow.com/questions/13257571/call-command-vs-start-with-wait-option
So the main difference is how we pass down the variables, I believe it's not an issue here.